stripe_official.php

```diff
@@ -892,7 +892,7 @@ protected function retrievePaymentIntent($amount, $currency)
                 }
 
                 // Check that the currency is still correct
-                if ($intent->currency != $currency) {
+                if ($intent->currency !=  Tools::strtolower($currency)) {
```

Comment:
Could you please remove the double space before ` Tools::strtolower` ?
```suggestion
               if ($intent->currency != Tools::strtolower($currency)) {
```